src/Lesson7/Les7_1.java

```diff
@@ -0,0 +1,29 @@
+package Lesson7;
+
+public class Les7_1 {
+    public static void arrayDivision(int[] array, int divisor) {
+        if (divisor == 0) {
```

Comment:
your implementation is also possible but it's a good idea to handle a possible Arithmetic exception through try/catch block